docs/management/admin/host-isolation-exceptions.asciidoc

```diff
@@ -0,0 +1,41 @@
+[[host-isolation-exceptions]]
+[chapter]
+= Host isolation exceptions
+
+You can configure host isolation exceptions for specific IP addresses that <<host-isolation-ov, isolated hosts>> are still allowed to communicate with, even when blocked from the rest of your network. Isolated hosts can still send data to {es} and {kib}, so you don't need to set up host isolation exceptions for them.
+
+Host isolation exceptions support IPv4 addresses, with optional CIDR (classless inter-domain routing) notation.
+
+IMPORTANT: Each host isolation exception IP address should be a highly trusted and secure location, since you’re allowing it to communicate with hosts that you've isolated to contain a potential threat and prevent it from spreading.
```

Comment:
```suggestion
IMPORTANT: Each host isolation exception IP address should be a highly trusted and secure location since you’re allowing it to communicate with hosts that you've isolated to contain a potential threat and prevent it from spreading.
```

Reply:
Can the second part of this sentence be shortened at all? Maybe to something like "since you're allowing it to communicate with potentially dangerous hosts that you've isolated."

Reply:
@nastasha-solomon Yeah, it is a long sentence. Earlier, @kevinlog and I revised this specifically to convey the idea of limiting a spreading threat, so instead of omitting that info, maybe tighten up the phrasing?
```suggestion
IMPORTANT: Each host isolation exception IP address should be a highly trusted and secure location since you’re allowing it to communicate with hosts that you've isolated to prevent a potential threat from spreading.
```

Reply:
Your revision looks good! The only tweak I'd suggested is to change "you've isolated" to "are isolated" or "have been isolated" because anyone could've isolated the host.

Reply:
Good point; updated and committed above!